bay/images/docker/settings.php

```diff
@@ -304,25 +304,32 @@
 $config['clamav.settings']['mode_daemon_tcpip']['port'] = $clamav_port;
 
 // Configure elasticsearch connections from environment variables.
-if (!in_array($lagoon_env_type, ['local', 'ci'])) {
-  if (getenv('SEARCH_HASH') && getenv('SEARCH_URL')) {
-    $config['elasticsearch_connector.cluster.elasticsearch_bay']['url'] = sprintf('http://%s.%s', getenv('SEARCH_HASH'), getenv('SEARCH_URL'));
-  }
+if (getenv('SEARCH_HASH') && getenv('SEARCH_URL')) {
+  $config['elasticsearch_connector.cluster.elasticsearch_bay']['url'] = sprintf('http://%s.%s', getenv('SEARCH_HASH'), getenv('SEARCH_URL'));
+} else {
+  $config['elasticsearch_connector.cluster.elasticsearch_bay']['url'] =  "http://elasticsearch:9200";
+}
 
-  if (getenv('SEARCH_INDEX')) {
-    $config['elasticsearch_connector.cluster.elasticsearch_bay']['options']['rewrite']['rewrite_index'] = 1;
-    $config['elasticsearch_connector.cluster.elasticsearch_bay']['options']['rewrite']['index'] = [
-      'prefix' => getenv('SEARCH_INDEX'),
-      'suffix' => '',
-    ];
-  }
+if (getenv('SEARCH_INDEX')) {
+  $config['elasticsearch_connector.cluster.elasticsearch_bay']['options']['rewrite']['rewrite_index'] = 1;
+  $config['elasticsearch_connector.cluster.elasticsearch_bay']['options']['rewrite']['index'] = [
+    'prefix' => getenv('SEARCH_INDEX'),
+    'suffix' => '',
+  ];
+} else {
+  $config['elasticsearch_connector.cluster.elasticsearch_bay']['options']['rewrite']['index'] = [
+    'prefix' => 'elasticsearch_index_nonprod_',
```

Comment:
Lets change this value to something that works as a generic default value for nonprod and prod.

Perhaps `elasticsearch_index_default_` ?

Reply:
@nicksantamaria I've added the change to a more sane default value but before marking this as 'Needs Review' I'll  put up a corresponding CMDB PR to get this default set for projects.

Reply:
Just clarifying this is doing away with the naming convention of `elasticsearch_index_pr` and replacing it with the suggested default.

Reply:
@nicksantamaria I've updated CMDB values in this PR https://github.com/dpc-sdp/sdp-cmdb/pull/542, once that's deployed this should be safe to merge.